### PR TITLE
cleanup: cleanup azurecns

### PR DESCRIPTION
**Pull request by AlisonB319**
**What type of PR is this?**
/kind cleanup
cleaning up older versions of azurecns

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:

```
none
```

### CODE REVIEW

File: parts/linux/cloud-init/artifacts/components.json [hunk 1]
```diff
@@ -71,18 +71,10 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-cns:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.4.52",
```
Comment (rbtr) [inline parts/linux/cloud-init/artifacts/components.json:74]: 1.4.52 is still used in AKS (for k8s 1.27)

File: parts/linux/cloud-init/artifacts/components.json [hunk 2]
```diff
@@ -71,18 +71,10 @@
...
-        "v1.5.26",
```
Comment (rbtr) [inline parts/linux/cloud-init/artifacts/components.json:75]: 1.5.26 is also still used in AKS (for k8s 1.28, until we finish rolling out 1.5.32)

**Review by rbtr — CHANGES_REQUESTED** [review]

File: parts/linux/cloud-init/artifacts/components.json (hunk 2 above)
Comment (AlisonB319) [reply parts/linux/cloud-init/artifacts/components.json:75]: Sounds good, I'll leave this PR open for when we are ready to remove 1.5.26

Comment (rbtr) [reply parts/linux/cloud-init/artifacts/components.json:75]: 1.5.32 has started rolling out to AKS prod so we can follow up here in ~2 weeks

**Comment by cameronmeissner** [comment]
I believe this was addressed in a separate PR

**Pull request closed without merging**